src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+public class Task {
+    protected String description;
+
+
+    public Task(String description) {
+        this.description = description;
+
+    }
+    public String getStatusIcon() {
+        return "";
+    }
+    public String getTypeIcon() {
+        return "";
+    }
+    public void setDone(boolean b) {
```

Comment:
Unsure of the use of this empty function

Reply:
This is for overriding